composer/callbacks/mlperf.py

```diff
@@ -250,7 +250,9 @@ def _get_dataloader_stats(self, dataloader: Iterable):
             import ffcv  # type: ignore
 
             if isinstance(dataloader, ffcv.loader.Loader):
-                return (dataloader.batch_size, len(dataloader) * dataloader.batch_size)  # type: ignore
+                return (
+                    dataloader.batch_size,  # type: ignore
```

Comment:
Curious why do we need type ignores?

Reply:
This was there before, but I think it can't find the `batch_size` attribute since `dataloader` is typed as an iterable.

Reply:
hmm, the isinstance check should tell it that it is a ffcv dataloader and not just an iterable. Not a big deal, but could you paste the pyright error in a comment?

Reply:
Sure would be helpful to understand.  This is the pyright failure:
```
Found 359 source files
/Users/bandishshah/composer/composer/callbacks/mlperf.py
  /Users/bandishshah/composer/composer/callbacks/mlperf.py:254:36 - error: Cannot access member "batch_size" for type "Iterable[Unknown]"
    Member "batch_size" is unknown (reportGeneralTypeIssues)
  /Users/bandishshah/composer/composer/callbacks/mlperf.py:254:63 - error: Cannot access member "next_traversal_order" for type "Iterable[Unknown]"
    Member "next_traversal_order" is unknown (reportGeneralTypeIssues)
2 errors, 0 warnings, 0 informations
Completed in 9.713sec
```